### ISSUE TITLE
Sending attachments is not working

### ISSUE THREAD

**Issue by RodolfoSilva** [elixir-saas/resend-elixir#3]
I'm trying to send an attachment. It is available when you check the local mailbox, but it is not available in production after switching to resend.

```ex
        attachment =
          Swoosh.Attachment.new({:data, Base.decode64!(blob)},
            filename: "appointments.pdf",
            content_type: "application/pdf"
          )

        logo_attachment =
          Swoosh.Attachment.new(
            :code.priv_dir(:my_app)
            |> Path.join(["static", "/logo.png"]),
            filename: "logo.png",
            type: :inline
          )

        Swoosh.Email.new()
        |> Swoosh.Email.to({"José", "jose@elixir-lang.com"})
        |> Swoosh.Email.from({"Example", "no-reply@my-domain.com"})
        |> Swoosh.Email.reply_to({"Example", "reply@my-domain.com"})
        |> Swoosh.Email.subject("Welcome")
        |> Swoosh.Email.html_body("Contnent <img src=\"cid:logo.png\" />")
        |> Swoosh.Email.text_body(
          "My Content"
        )
        |> Swoosh.Email.attachment(logo_attachment)
        |> Swoosh.Email.attachment(attachment)
        |> MyApp.Mailer.deliver()
```

This is the log from resend panel:

<img width="898" alt="image" src="https://github.com/user-attachments/assets/d57d3ad4-9f6e-45ce-b215-56a413d5a729">

It seems the attachment is not included.

**Comment by RodolfoSilva** [comment]
@jtormey, I noticed the release that support this feature hasn't been deployed yet (#2). Do you have any plans to release a new version?

**Comment by jtormey** [comment]
You're right, that change hadn't been published yet. Thank you for the reminder!

[`v0.4.3`](https://hexdocs.pm/resend/0.4.3) is now live. Could you install it and try to send an attachment again?

**Comment by RodolfoSilva** [comment]
@jtormey I've found a bug, I'm submitting a PR with the fix.

**Comment by RodolfoSilva** [comment]
@jtormey done: https://github.com/elixir-saas/resend-elixir/pull/4, this will solve the bug!

**Comment by jtormey** [comment]
Merged and published in v0.4.4, thanks!

**Comment by RodolfoSilva** [comment]
Thanks!
